src/Files.App/Data/Contexts/MultiPanes/IMultiPanesContext.cs

```diff
@@ -18,6 +18,11 @@ public interface IMultiPanesContext
 		/// </summary>
 		event EventHandler? ActivePaneChanged;
 
+		/// <summary>
+		/// Gets invoked when shell pane arrangement is changed.
```

Comment:
```suggestion
		/// Invoked when shell pane arrangement is changed.
```

Reply:
It's out of scope for now but I'd like tospecify our guideline for event comments in order to keep consistency.

WinAppSdk API starts with `Occurs during ...`, while WCT API starts with `Fires when ...`
![image](https://github.com/files-community/Files/assets/62196528/6b1d6b25-7383-4ac6-a312-687b8c765f0a)

Reference: https://github.com/search?q=repo%3Afiles-community%2FFiles+%22Gets+invoked%22&type=code

Which one would you like or would you like to get it started with `Invoked ...`?

Reply:
`Occurs` or `Invoked` is fine with me.